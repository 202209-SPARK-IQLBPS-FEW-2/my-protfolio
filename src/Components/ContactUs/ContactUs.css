@media screen and (min-width: 480px) {
    .one {
      width: 128px;
    }
  }