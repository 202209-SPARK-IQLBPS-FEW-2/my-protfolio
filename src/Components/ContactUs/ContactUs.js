import React from "react";
import "./ContactUs.css";
// import { Link } from "react-router-dom";


export default function ContactUs() {
  return (
<p>ContactUs </p>
       )}