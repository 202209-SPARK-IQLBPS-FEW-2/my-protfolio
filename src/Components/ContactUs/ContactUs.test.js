import React from "react";
import { screen ,render } from "@testing-library/react";
import ContactUs from "./ContactUs";
import userEvent from '@testing-library/user-event';
import renderer from "react-test-renderer";



describe("contact component", ()=>{
  //snapshot
it ("render first header",()=>{
  const fa = renderer.create(<ContactUs title ={"SEND US YOUR REQUEST"}/>).toJSON();
  expect(fa).toMatchSnapshot();
  
})



it ("second header",()=>{
  const fa = renderer.create(<ContactUs title ={"Do you have a question"}/>).toJSON();
  expect(fa).toMatchSnapshot();
  
})

it ("radio button title",()=>{
  const fa = renderer.create(<ContactUs title ={"Type of contact"}/>).toJSON();
  expect(fa).toMatchSnapshot();
  
})

it ("contact information",()=>{
  const fa = renderer.create(<ContactUs title ={"Find US At:"}/>).toJSON();
  expect(fa).toMatchSnapshot();
  
})

//get By Text

it("address line 1", async() => {
  render(
     <ContactUs title ="Nergiz Plaza" />
 );
const headingElement = screen.getByText(/Nergiz Plaza/i);
 expect(headingElement).toBeInTheDocument();
});


//find by
it("address line 2", async() => {
  render(
     <ContactUs title ="3rd Floor" />
 );
const headingElement = await screen.findByText(/3rd Floor/i);
 expect(headingElement).toBeInTheDocument();
});

//query by
it("address line 2", async() => {
  render(
     <ContactUs title ="3rd Floor" />
 );
const headingElement =  screen.queryByText(/call me /i);
 expect(headingElement).not.toBeInTheDocument();
});

//get by title
it("address line 3", async() => {
  render(
     <ContactUs title ="Bakhtiyari Street 40m" />
 );


const headingElement = screen.getByTitle("Street");
 expect(headingElement).toBeInTheDocument();
});
it("City/Country", async() => {
  render(
     <ContactUs title ="Erbil, Iraq" />
 );
const headingElement = screen.getByText(/Erbil, Iraq/i);
 expect(headingElement).toBeInTheDocument();
});

//get by role
it("Zip code", async() => {
  render(
     <ContactUs title ="44001" />
 );
const headingElement = screen.getByRole("heading", {name: "44001"});
 expect(headingElement).toBeInTheDocument();
});

//show error when all fields are not entered
it("should show error message when all the fields are not entered", () => {
  render(<ContactUs />);
  const buttonElement = screen.getByRole("button");
  userEvent.click(buttonElement);
});
});
