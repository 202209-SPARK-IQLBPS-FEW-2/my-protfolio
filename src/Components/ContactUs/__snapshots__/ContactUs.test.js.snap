// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact component contact information 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-10/12	"
    >
      <h1
        className="	w-10/12 mt-10 font-normal font-sans text-2xl  md:text-3xl"
      >
        SEND US YOUR REQUEST
      </h1>
      <h1
        className=" mt-5 text-zinc-500 font-sans text-s sm:text-sm md:text-xl"
      >
        Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we will be happy to help!
      </h1>
    </div>
    <div
      className="flex flex-row w-10/12	 mt-20 gap-56"
    >
      <div
        className=" w-2/4 mt-6"
      >
        <h1
          className="text-s sm:text-sm md:text-2xl font-medium	"
        >
          Type of contact
        </h1>
        <div
          className="flex flex-col mt-5"
        >
          <label
            htmlFor="ques1"
          >
            I have a question about the service.
            <input
              className="float-left mr-4 mt-2 md-mr-2 md-mt-0 md-float-left"
              id="ques1"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques2"
          >
            I am a registered client and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques2"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques3"
          >
            I am a counselor interested in joining.
            <input
              className="float-left mr-4 mt-2"
              id="ques3"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques4"
          >
            I am a registered counselor and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques4"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques5"
          >
            I have a business-related inquiry.
            <input
              className="float-left mr-4 mt-2"
              id="ques5"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques6"
          >
            I am interested in Healing Online for my organization.
            <input
              className="float-left mr-4 mt-2"
              id="ques6"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques7"
          >
            I have a billing related question.
            <input
              className="float-left mr-4 mt-2 mt-2"
              id="ques7"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
        </div>
      </div>
      <div
        className=""
      >
        <img
          alt=""
          className="sm:w-auto w-auto h-auto "
          src="Group.png"
        />
      </div>
    </div>
  </div>
  <div
    className="flex flex-row justify-center items-center my-12"
  >
    <form
      className="w-10/12  "
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col sm:w-2/4 w-2/4 justify-center  items-start space-y-5  sm:px-28 px-48"
      >
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="name"
        >
          Full Name:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4	px-4"
            name="name"
            onChange={[Function]}
            placeholder="Enter your full name here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl "
          htmlFor="email"
        >
          Email:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4 px-4 "
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="email"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="detail"
        >
          Details:
          <input
            className=" rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm  placeholder border my-4 px-4"
            name="details"
            onChange={[Function]}
            placeholder="Enter your details here..."
            style={
              Object {
                "height": "180px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <button
          aria-label="Send"
          className="w-28 sm:w-56	 h-8 sm:h-16 rounded-md font-normal text-l sm:text-2xl mt-20"
          style={
            Object {
              "background": "#2DD3E3",
            }
          }
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
    <div
      className="flex w-56 justify-center items-center mx-20 mt-96 sm:my-48"
    >
      <div
        className="flex flex-col rounded-lg w-96	justify-center items-start text-s sm:text-2xl"
        style={
          Object {
            "background": "#EAF8F9",
            "borderRadius": "35px",
          }
        }
      >
        <h1
          className="ml-5 my-2 font-normal"
        >
          Find US At:
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Nergiz Plaza
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          3rd Floor
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
          title="Street"
        >
          Bakhtiyari Street 40m
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Erbil, Iraq
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          44001
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`contact component radio button title 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-10/12	"
    >
      <h1
        className="	w-10/12 mt-10 font-normal font-sans text-2xl  md:text-3xl"
      >
        SEND US YOUR REQUEST
      </h1>
      <h1
        className=" mt-5 text-zinc-500 font-sans text-s sm:text-sm md:text-xl"
      >
        Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we will be happy to help!
      </h1>
    </div>
    <div
      className="flex flex-row w-10/12	 mt-20 gap-56"
    >
      <div
        className=" w-2/4 mt-6"
      >
        <h1
          className="text-s sm:text-sm md:text-2xl font-medium	"
        >
          Type of contact
        </h1>
        <div
          className="flex flex-col mt-5"
        >
          <label
            htmlFor="ques1"
          >
            I have a question about the service.
            <input
              className="float-left mr-4 mt-2 md-mr-2 md-mt-0 md-float-left"
              id="ques1"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques2"
          >
            I am a registered client and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques2"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques3"
          >
            I am a counselor interested in joining.
            <input
              className="float-left mr-4 mt-2"
              id="ques3"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques4"
          >
            I am a registered counselor and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques4"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques5"
          >
            I have a business-related inquiry.
            <input
              className="float-left mr-4 mt-2"
              id="ques5"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques6"
          >
            I am interested in Healing Online for my organization.
            <input
              className="float-left mr-4 mt-2"
              id="ques6"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques7"
          >
            I have a billing related question.
            <input
              className="float-left mr-4 mt-2 mt-2"
              id="ques7"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
        </div>
      </div>
      <div
        className=""
      >
        <img
          alt=""
          className="sm:w-auto w-auto h-auto "
          src="Group.png"
        />
      </div>
    </div>
  </div>
  <div
    className="flex flex-row justify-center items-center my-12"
  >
    <form
      className="w-10/12  "
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col sm:w-2/4 w-2/4 justify-center  items-start space-y-5  sm:px-28 px-48"
      >
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="name"
        >
          Full Name:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4	px-4"
            name="name"
            onChange={[Function]}
            placeholder="Enter your full name here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl "
          htmlFor="email"
        >
          Email:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4 px-4 "
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="email"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="detail"
        >
          Details:
          <input
            className=" rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm  placeholder border my-4 px-4"
            name="details"
            onChange={[Function]}
            placeholder="Enter your details here..."
            style={
              Object {
                "height": "180px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <button
          aria-label="Send"
          className="w-28 sm:w-56	 h-8 sm:h-16 rounded-md font-normal text-l sm:text-2xl mt-20"
          style={
            Object {
              "background": "#2DD3E3",
            }
          }
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
    <div
      className="flex w-56 justify-center items-center mx-20 mt-96 sm:my-48"
    >
      <div
        className="flex flex-col rounded-lg w-96	justify-center items-start text-s sm:text-2xl"
        style={
          Object {
            "background": "#EAF8F9",
            "borderRadius": "35px",
          }
        }
      >
        <h1
          className="ml-5 my-2 font-normal"
        >
          Find US At:
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Nergiz Plaza
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          3rd Floor
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
          title="Street"
        >
          Bakhtiyari Street 40m
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Erbil, Iraq
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          44001
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`contact component render first header 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-10/12	"
    >
      <h1
        className="	w-10/12 mt-10 font-normal font-sans text-2xl  md:text-3xl"
      >
        SEND US YOUR REQUEST
      </h1>
      <h1
        className=" mt-5 text-zinc-500 font-sans text-s sm:text-sm md:text-xl"
      >
        Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we will be happy to help!
      </h1>
    </div>
    <div
      className="flex flex-row w-10/12	 mt-20 gap-56"
    >
      <div
        className=" w-2/4 mt-6"
      >
        <h1
          className="text-s sm:text-sm md:text-2xl font-medium	"
        >
          Type of contact
        </h1>
        <div
          className="flex flex-col mt-5"
        >
          <label
            htmlFor="ques1"
          >
            I have a question about the service.
            <input
              className="float-left mr-4 mt-2 md-mr-2 md-mt-0 md-float-left"
              id="ques1"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques2"
          >
            I am a registered client and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques2"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques3"
          >
            I am a counselor interested in joining.
            <input
              className="float-left mr-4 mt-2"
              id="ques3"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques4"
          >
            I am a registered counselor and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques4"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques5"
          >
            I have a business-related inquiry.
            <input
              className="float-left mr-4 mt-2"
              id="ques5"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques6"
          >
            I am interested in Healing Online for my organization.
            <input
              className="float-left mr-4 mt-2"
              id="ques6"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques7"
          >
            I have a billing related question.
            <input
              className="float-left mr-4 mt-2 mt-2"
              id="ques7"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
        </div>
      </div>
      <div
        className=""
      >
        <img
          alt=""
          className="sm:w-auto w-auto h-auto "
          src="Group.png"
        />
      </div>
    </div>
  </div>
  <div
    className="flex flex-row justify-center items-center my-12"
  >
    <form
      className="w-10/12  "
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col sm:w-2/4 w-2/4 justify-center  items-start space-y-5  sm:px-28 px-48"
      >
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="name"
        >
          Full Name:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4	px-4"
            name="name"
            onChange={[Function]}
            placeholder="Enter your full name here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl "
          htmlFor="email"
        >
          Email:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4 px-4 "
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="email"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="detail"
        >
          Details:
          <input
            className=" rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm  placeholder border my-4 px-4"
            name="details"
            onChange={[Function]}
            placeholder="Enter your details here..."
            style={
              Object {
                "height": "180px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <button
          aria-label="Send"
          className="w-28 sm:w-56	 h-8 sm:h-16 rounded-md font-normal text-l sm:text-2xl mt-20"
          style={
            Object {
              "background": "#2DD3E3",
            }
          }
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
    <div
      className="flex w-56 justify-center items-center mx-20 mt-96 sm:my-48"
    >
      <div
        className="flex flex-col rounded-lg w-96	justify-center items-start text-s sm:text-2xl"
        style={
          Object {
            "background": "#EAF8F9",
            "borderRadius": "35px",
          }
        }
      >
        <h1
          className="ml-5 my-2 font-normal"
        >
          Find US At:
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Nergiz Plaza
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          3rd Floor
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
          title="Street"
        >
          Bakhtiyari Street 40m
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Erbil, Iraq
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          44001
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`contact component renders 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-10/12	"
    >
      <h1
        className="	w-10/12 mt-10 font-normal font-sans text-2xl  md:text-3xl"
      >
        SEND US YOUR REQUEST
      </h1>
      <h1
        className=" mt-5 text-zinc-500 font-sans text-s sm:text-sm md:text-xl"
      >
        Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we will be happy to help!
      </h1>
    </div>
    <div
      className="flex flex-row w-10/12	 mt-20 gap-56"
    >
      <div
        className=" w-2/4 mt-6"
      >
        <h1
          className="text-s sm:text-sm md:text-2xl font-medium	"
        >
          Type of contact
        </h1>
        <div
          className="flex flex-col mt-5"
        >
          <label
            htmlFor="ques1"
          >
            I have a question about the service.
            <input
              className="float-left mr-4 mt-2 md-mr-2 md-mt-0 md-float-left"
              id="ques1"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques2"
          >
            I am a registered client and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques2"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques3"
          >
            I am a counselor interested in joining.
            <input
              className="float-left mr-4 mt-2"
              id="ques3"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques4"
          >
            I am a registered counselor and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques4"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques5"
          >
            I have a business-related inquiry.
            <input
              className="float-left mr-4 mt-2"
              id="ques5"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques6"
          >
            I am interested in Healing Online for my organization.
            <input
              className="float-left mr-4 mt-2"
              id="ques6"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques7"
          >
            I have a billing related question.
            <input
              className="float-left mr-4 mt-2 mt-2"
              id="ques7"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
        </div>
      </div>
      <div
        className=""
      >
        <img
          alt=""
          className="sm:w-auto w-auto h-auto "
          src="Group.png"
        />
      </div>
    </div>
  </div>
  <div
    className="flex flex-row justify-center items-center my-12"
  >
    <form
      className="w-10/12  "
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col sm:w-2/4 w-2/4 justify-center  items-start space-y-5  sm:px-28 px-48"
      >
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="name"
        >
          Full Name:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4	px-4"
            name="name"
            onChange={[Function]}
            placeholder="Enter your full name here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl "
          htmlFor="email"
        >
          Email:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4 px-4 "
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="email"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="detail"
        >
          Details:
          <input
            className=" rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm  placeholder border my-4 px-4"
            name="details"
            onChange={[Function]}
            placeholder="Enter your details here..."
            style={
              Object {
                "height": "180px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <button
          aria-label="Send"
          className="w-28 sm:w-56	 h-8 sm:h-16 rounded-md font-normal text-l sm:text-2xl mt-20"
          style={
            Object {
              "background": "#2DD3E3",
            }
          }
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
    <div
      className="flex w-56 justify-center items-center mx-20 mt-96 sm:my-48"
    >
      <div
        className="flex flex-col rounded-lg w-96	justify-center items-start text-s sm:text-2xl"
        style={
          Object {
            "background": "#EAF8F9",
            "borderRadius": "35px",
          }
        }
      >
        <h1
          className="ml-5 my-2 font-normal"
        >
          Find US At:
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Nergiz Plaza
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          3rd Floor
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
          title="Street"
        >
          Bakhtiyari Street 40m
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Erbil, Iraq
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          44001
        </h1>
      </div>
    </div>
  </div>
</div>
`;

exports[`contact component second header 1`] = `
<div>
  <div
    className="flex flex-wrap justify-center items-center"
  >
    <div
      className="w-10/12	"
    >
      <h1
        className="	w-10/12 mt-10 font-normal font-sans text-2xl  md:text-3xl"
      >
        SEND US YOUR REQUEST
      </h1>
      <h1
        className=" mt-5 text-zinc-500 font-sans text-s sm:text-sm md:text-xl"
      >
        Do you have a question, concern, idea, feedback, or problem? If you need assistance, please fill out the form below and we will be happy to help!
      </h1>
    </div>
    <div
      className="flex flex-row w-10/12	 mt-20 gap-56"
    >
      <div
        className=" w-2/4 mt-6"
      >
        <h1
          className="text-s sm:text-sm md:text-2xl font-medium	"
        >
          Type of contact
        </h1>
        <div
          className="flex flex-col mt-5"
        >
          <label
            htmlFor="ques1"
          >
            I have a question about the service.
            <input
              className="float-left mr-4 mt-2 md-mr-2 md-mt-0 md-float-left"
              id="ques1"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques2"
          >
            I am a registered client and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques2"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques3"
          >
            I am a counselor interested in joining.
            <input
              className="float-left mr-4 mt-2"
              id="ques3"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques4"
          >
            I am a registered counselor and I need support.
            <input
              className="float-left mr-4 mt-2"
              id="ques4"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques5"
          >
            I have a business-related inquiry.
            <input
              className="float-left mr-4 mt-2"
              id="ques5"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques6"
          >
            I am interested in Healing Online for my organization.
            <input
              className="float-left mr-4 mt-2"
              id="ques6"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
          <label
            htmlFor="ques7"
          >
            I have a billing related question.
            <input
              className="float-left mr-4 mt-2 mt-2"
              id="ques7"
              name="questions"
              onChange={[Function]}
              type="radio"
              value={true}
            />
          </label>
          <br />
        </div>
      </div>
      <div
        className=""
      >
        <img
          alt=""
          className="sm:w-auto w-auto h-auto "
          src="Group.png"
        />
      </div>
    </div>
  </div>
  <div
    className="flex flex-row justify-center items-center my-12"
  >
    <form
      className="w-10/12  "
      onSubmit={[Function]}
    >
      <div
        className="flex flex-col sm:w-2/4 w-2/4 justify-center  items-start space-y-5  sm:px-28 px-48"
      >
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="name"
        >
          Full Name:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4	px-4"
            name="name"
            onChange={[Function]}
            placeholder="Enter your full name here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl "
          htmlFor="email"
        >
          Email:
          <input
            className="rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm placeholder border my-4 px-4 "
            name="email"
            onChange={[Function]}
            placeholder="Enter your email address here..."
            style={
              Object {
                "height": "68px",
                "width": "604px",
              }
            }
            type="email"
          />
        </label>
        <label
          className="font-normal text-l sm:text-sm md:text-xl"
          htmlFor="detail"
        >
          Details:
          <input
            className=" rounded-lg shadow-md sm:placeholder:text-lg placeholder:text-sm  placeholder border my-4 px-4"
            name="details"
            onChange={[Function]}
            placeholder="Enter your details here..."
            style={
              Object {
                "height": "180px",
                "width": "604px",
              }
            }
            type="text"
          />
        </label>
        <button
          aria-label="Send"
          className="w-28 sm:w-56	 h-8 sm:h-16 rounded-md font-normal text-l sm:text-2xl mt-20"
          style={
            Object {
              "background": "#2DD3E3",
            }
          }
          type="submit"
        >
          SUBMIT
        </button>
      </div>
    </form>
    <div
      className="flex w-56 justify-center items-center mx-20 mt-96 sm:my-48"
    >
      <div
        className="flex flex-col rounded-lg w-96	justify-center items-start text-s sm:text-2xl"
        style={
          Object {
            "background": "#EAF8F9",
            "borderRadius": "35px",
          }
        }
      >
        <h1
          className="ml-5 my-2 font-normal"
        >
          Find US At:
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Nergiz Plaza
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          3rd Floor
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
          title="Street"
        >
          Bakhtiyari Street 40m
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          Erbil, Iraq
        </h1>
        <h1
          className="ml-5 font-light	text-gray-500		"
        >
          44001
        </h1>
      </div>
    </div>
  </div>
</div>
`;
